sdk/core/azure-core/inc/azure/core/internal/json/json_optional.hpp

```diff
@@ -71,4 +94,29 @@ namespace Azure { namespace Core { namespace Json { namespace _internal {
     }
   };
 
+  template <class T, class R>
+  static inline void SetFromNullable(
```

Comment:
This method is specific for writing a json key is from a Nullable<T>.  This is to avoid checking if the Nullable<T> contains a value or not everywhere when serializing json